File: 027_remove_element.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        if (nums.size()==0) return 0;
        int count=0;
        for (int i=0;i<nums.size();i++) {
            if (nums[i] == val) continue;
            else 
            {nums[count]=nums[i];
            count++;}
        }
        nums.resize(count);
        return count;
        }
};

File: algorithms/python3/027_remove_element.py
class Solution:
    def removeElement(self, nums, val):
        i=0
        for j in range(len(nums)):
            if nums[j] != val:
                nums[i]=nums[j]
                i+=1
        return i


nums=[0,1,2,2,3,0,4,2]
a=Solution()

print(a.removeElement(nums,2))


